tools/gendocs/gendocs/gendocs.go

```diff
@@ -77,6 +78,11 @@ func extractExamples(cmd *cobra.Command) Examples {
 		o := &unstructured.Unstructured{
 			Object: make(map[string]interface{}),
 		}
+		if strings.HasPrefix(cmd.CommandPath(), "oc adm") {
```

Comment:
You can write this simply as:
```golang
o.Object["isAdminCommand"] = strings.hasPrefix(cmd.CommandPath(), "oc adm") 
```

Reply:
As requested in slack, I'll make this update in a future PR.